src/openrct2/ride/TrackData.h

```diff
@@ -81,13 +82,30 @@ namespace OpenRCT2::TrackMetaData
         WoodenSupportTransitionType transitionType = WoodenSupportTransitionType::None;
     };
 
+    struct SequenceMetalSupport
+    {
+        MetalSupportPlace place = MetalSupportPlace::None;
+        uint8_t alternates = false;
+    };
+
+    struct SequenceDescriptor
+    {
+        SequenceClearance clearance{};
```

Comment:
It appears these properties could be inlined into Sequence Descriptor. Something for a followup PR if true.

Reply:
Suggestions for a better name are welcomed.